AWSCognitoIdentityProvider/AWSCognitoIdentityUser.m

```diff
@@ -1488,6 +1488,28 @@ -(BOOL) isSignedIn {
     return refreshToken != nil;
 }
 
+- (AWSTask<AWSCognitoIdentityProviderRevokeTokenResponse *> *) revokeToken {
+    __block NSString * keyChainNamespace = [self keyChainNamespaceClientId];
+    NSString * accessTokenKey = [self keyChainKey:keyChainNamespace key:AWSCognitoIdentityUserAccessToken];
+    NSString * accessTokenString = self.pool.keychain[accessTokenKey];
+    AWSCognitoIdentityUserSessionToken * accessToken = [[AWSCognitoIdentityUserSessionToken alloc] initWithToken:accessTokenString];
+    if ([accessToken.tokenClaims objectForKey:@"origin_jti"] == nil) {
+        AWSDDLogVerbose(@"Access Token does not contain `origin_jti` claim. Skip revoking tokens.");
+        return [AWSTask taskWithResult:nil];
+    }
+    
+    AWSCognitoIdentityProviderRevokeTokenRequest *request = [AWSCognitoIdentityProviderRevokeTokenRequest new];
+    NSString * refreshToken = [self refreshTokenFromKeyChain:keyChainNamespace];
+    request.token = refreshToken;
+    request.clientId = self.pool.userPoolConfiguration.clientId;
+    request.clientSecret = self.pool.userPoolConfiguration.clientSecret;
+    
+    return [[self.pool.client revokeToken:request] continueWithSuccessBlock:^id _Nullable(AWSTask<AWSCognitoIdentityProviderRevokeTokenResponse *> * _Nonnull task) {
+        AWSDDLogVerbose(@"Revoke token completed.");
+        return task;
+    }];
```

Comment:
```suggestion
    return [self.pool.client revokeToken:request];
```

Does this work?

Reply:
yup this works, the logging seems redundant, taking ur suggestion